### ISSUE TITLE
Intellisence not working in VSCode PowerShell Extension 21.2.2 on Windows Server 2019 Datacenter Version 1809

### ISSUE THREAD

**Issue by PaulTheScott** [PowerShell/vscode-powershell#3240]
I same issue that was identified Issue #179. 
(https://github.com/PowerShell/vscode-powershell/issues/179) - Or perhaps something is causing the error.

Earlier today, I installed latest versions of VSCode and PowerShell.
OS is Windows Server 2019 Datacenter and all updates are installed current.

Last version of .NET was installed this morning.

All versions of the software are x64.

In the mean time I will also try out the PowerShell Preview plug-in and see if it works there.

I also followed the steps here https://github.com/PowerShell/vscode-powershell/issues/179 to get the old PowerShell ISE to work and pick up the intelligence for PowerShell 7.1.3 (64-bit).

Any help on what I might be doing wrong would be greatly appreciated. Thanks!

**Comment by PaulTheScott** [comment]
I have found what was causing my intellisence feature problems in reported in issue #3240.

Issue: Default Intellisence setting are missing from settings.json in VSCode.
Fix: Provided on this webpage: https://code.visualstudio.com/docs/editor/intellisense
Problem Resolved: Yes.

I have followed the instructions here (https://code.visualstudio.com/docs/editor/intellisense) and have added the missing settings to the settings.json file. I then restarted VSCode and found all Intellisence are now working for my PowerShell Extension.

Thank you for providing such a wonderful tool to the world for free. Not everyone can afford to pay extensive license costs involved with proprietary software vendors.
